packages/material-ui/src/internal/svg-icons/createIcon.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SvgIcon from '../../SvgIcon';
+
+export default function createIcon(name, path) {
```

Comment:
What do you think of `createSvgIcon` for consistency with `@material-ui/icons`'s helper?

Reply:
This is the helper I'm referring to, we can't use for circular dependency constraints, but better be consistent.
```jsx
import React from 'react';
import SvgIcon from '@material-ui/core/SvgIcon';

function createSvgIcon(path, displayName) {
  let Icon = props => (
    <SvgIcon {...props}>
      {path}
    </SvgIcon>
  );

  Icon.displayName = `${displayName}Icon`;
  Icon = React.memo(Icon);
  Icon.muiName = 'SvgIcon';

  return Icon;
};

export default createSvgIcon;
```

Reply:
We could just lift it into `@material-ui/utils` and import from there. It would also help consolidating all the utils that are scattered across packages.

Reply:
Sure, great idea.

Reply:
I'll do it in another PR with a breaking change though:

```diff
- import createSvgIcon from '@material-ui/icons/utils';
+ import { createSvgIcon } from '@material-ui/utils';
```